src/test/resources/pages/instructorFeedbackQuestionEditToTeamToTeam.html

```diff
@@ -738,11 +738,8 @@ <h5 class="col-xs-2" data-original-title="Select this option to use a custom tim
          </form>
       </div>
       <br/>
-      <div class="overflow-auto alert alert-success" id="statusMessage">
-         The question has been added to this feedback session.
+      <div id="statusMessage" style="display: none;">
```

Comment:
This may weaken this particular test. It will be better to craft a test for this issue that doesn't weaken the other existing tests.